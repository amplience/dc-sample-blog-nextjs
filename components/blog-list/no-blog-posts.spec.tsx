/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';
import NoBlogPosts from './no-blog-posts';

describe('NoBlogPosts', () => {
  it('should render component', async () => {
    const component = await renderer.create(<NoBlogPosts/>);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
