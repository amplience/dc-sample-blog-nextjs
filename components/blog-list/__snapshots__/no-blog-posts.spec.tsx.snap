// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoBlogPosts should render component 1`] = `
<div
  className="jsx-1210611463 no-published-blogs"
>
  <p
    className="jsx-1210611463"
  >
    No blogs published yet!
  </p>
</div>
`;
